# ATTRS{serial}=="FBL2XXX"
# ATTRS{idVendor}=="20d2"
# ATTRS{idProduct}=="5740"

port : "/dev/ttyMotor"
baud : 115200

frequencyH : 50   #higher frequency (value is in ms) #1/50ms = 20Hz
frequencyL : 100  #lower frequency
frequencyG : 100  #frequency for general queries

queryH:
 # motor_amps : ?A 
 # motor_command : ?M
 # hall_count : ?Cb
 # hall_speed : ?BS
 encoder_count : ?C
 encoder_speed : ?S  
 # volts : ?V
 # feedback : ?F 
 # battery_amps : ?BA
 # power : ?P          
# motor_amps2 : ?MA
# motor_command2 : ?FM
# hall_count2 : ?FM
# hall_speed2 : ?FS
# volts2 : ?V
# feedback2 : ?rmp 
# battery_amps2 : ?PHA
# power2 : ?sns          

# queryG: 
 #fault_flag : ?FF  
 # fault_flag2 : ?S  
# status_flag : ?FS
# firmware_id : ?FID  Users can add queries which do not require channel number under queryG tab. 

